python/tvm/relay/op/transform.py

```diff
@@ -1320,3 +1320,52 @@ def adv_index(inputs):
         Output tensor.
     """
     return _make.adv_index(Tuple(inputs))
+
+
+def sparsereshape(sparse_indices, sparse_values, prev_shape, new_shape):
+    """
+    Reshape a Sparse Tensor
```

Comment:
Could you note that this function only support tensors in COO format, not CSR. In other parts of the codebase, we tend to use CSR.

Reply:
Can you explain how this convention is different from the `sparse_to_dense` operator. I could only find that operator as an example of existing representations.

Reply:
The convention is the same as `sparse_to_dense`. However `sparse_dense` uses CSR and BSR formats. We should probably add documentation to `sparse_to_dense` too.